lib/board.rb

```diff
@@ -69,32 +72,18 @@ def collect_numbers(coordinates)
 
   def consecutive_letters?(coordinates)
     (collect_letters(coordinates)[0].ord..collect_letters(coordinates)[-1].ord).to_a == collect_letters(coordinates).map { |letter| letter.ord }
-    # do I need to sort for the first part?
-    # (letters[0].ord..letters[-1].ord).to_a == letters.map { |letter| letter.ord }
   end
 
   def consecutive_numbers?(coordinates)
     (collect_numbers(coordinates)[0]..collect_numbers(coordinates)[-1]).to_a == collect_numbers(coordinates)
-    # do I need to sort for the first part?
-    # (numbers[0]..numbers[-1]).to_a == numbers
   end
 
   def place(ship, coordinates)
-
     coordinates.each do |coordinate|
-      cell = cells[coordinate]
-      cell.place_ship(ship)
-
+      coordinate.place_ship(ship)
```

Comment:
it turned out to be even simpler. I used pry to find out what coordinate was and its a cell class so we can just call place_ship(ship) directly on it.